src/Makefile.am

```diff
@@ -53,9 +53,15 @@ libt8_installed_headers_forest = \
   src/t8_forest/t8_forest_profiling.h \
   src/t8_forest/t8_forest_io.h \
   src/t8_forest/t8_forest_adapt.h \
+  src/t8_forest/t8_forest_iterate.h src/t8_forest/t8_forest_partition.h \
+=======
   src/t8_forest/t8_forest_iterate.h \
   src/t8_forest/t8_forest_partition.h \
-  src/t8_forest/t8_forest_search/t8_forest_search.hxx
+  src/t8_forest/t8_forest_ghost_definition_wrapper.h \
+  src/t8_forest/t8_forest_ghost_definition.h \
+  src/t8_forest/t8_forest_ghost_definition.hxx \
+  src/t8_forest/t8_forest_search/t8_forest_search.hxx \
+  src/t8_forest/t8_forest_ghost_stencil.hxx
```

Comment:
Something went wrong with merge here.
"======="
and  some files are listed multiple times.